nilearn/_utils/cache_mixin.py

```diff
@@ -181,7 +181,7 @@ def cache(func, memory, func_memory_level=None, memory_level=None,
     func : function
         The function which output is to be cached.
 
-    memory : instance of joblib.Memory or string or pathlib.Path
+    memory : instance of joblib.Memory, string or pathlib.Path
```

Comment:
```suggestion
    memory : instance of :class:`joblib.Memory`, obj:`str` or :class:`pathlib.Path`
```

Reply:
I think it may be better to leave as is since the rest of the file does not include these roles. Also, I guess since it's under `_utils` module, the docstrings are not exposed.

Reply:
indeed if these docstrings are read directly in the source files rather than in rendered html documentation there is no need for markup